tests/integration/conftest.py

```diff
@@ -113,10 +113,14 @@ def pipfile(self):
 
     @property
     def lockfile(self):
-        p_path = os.sep.join([self.path, 'Pipfile.lock'])
+        p_path = self.lockfile_path
         with open(p_path, 'r') as f:
             return json.loads(f.read())
 
+    @property
+    def lockfile_path(self):
```

Comment:
This is a bit inconsistent because the pipfile equivalent is called `pipfile_location`

Reply:
In this particular class there is already `self.pipfile_path` above.  I'd think either leave this consistent within the class or change both?